Makefile

```diff
@@ -1,11 +1,10 @@
 .PHONY: docs flake8 example test coverage migrations
 
 docs:
-	cd docs; make html
-	open docs/_build/html/index.html
+	tox -e docs
 
 flake8:
-	tox -e flake8,isort
+	tox -e ruff,isort
```

Comment:
This isn't anything to do with documentation but it was right there looking at me so I fixed it